applications/matter_bridge/Kconfig.sysbuild

```diff
@@ -8,15 +8,42 @@
 config NRF_DEFAULT_IPC_RADIO
 	default y
 
+# Use HCI IPC as radio core image for nRF54H20 + nRF7002EK
+config NRF_DEFAULT_BLUETOOTH
+	default y if SOC_SERIES_NRF54HX && WIFI_NRF700X
+
+# Enable IEEE802.15.4 serialization to network core
+config NETCORE_IPC_RADIO_IEEE802154
```

Comment:
For now we can remove this line, as it applies only to 15.4 radio (Thread) and we support only Wi-Fi in this application (yet).